integreat_cms/cms/fixtures/test_data.json

```diff
@@ -379,12 +379,29 @@
       "message_partial_live_content_not_available": "Ein Teil von der Seite ist nicht in Ihrer Sprache. Der Teil ist aber in diesen Sprachen verfügbar:"
     }
   },
+  {
+    "model": "cms.language",
+    "pk": 7,
+    "fields": {
+      "slug": "hidden",
+      "bcp47_tag": "hid",
+      "native_name": "Hidden language",
+      "english_name": "Hidden language",
+      "text_direction": "LEFT_TO_RIGHT",
+      "primary_country_code": "hi",
```

Comment:
```suggestion
      "primary_country_code": "at",
```
`hi` is not part of the valid [choices](https://github.com/digitalfabrik/integreat-cms/blob/29313949a4756643590a83d93532c859bc4f269c/integreat_cms/cms/constants/countries.py#L10) for this model, so I suggest to use a different dummy-country-code.